### ISSUE TITLE
In the toolbar > add new > collection improperly indented 

### ISSUE THREAD

**Issue by pigeonflight** [plone/Products.CMFPlone#1446]
The word "Collection" runs over the collection icon, all the other icons and names are aligned properly
![image](https://cloud.githubusercontent.com/assets/31827/13592283/b03f7b6e-e4be-11e5-87e9-aba22238f143.png)

**Comment by drex44** [comment]
![snapshot1](https://cloud.githubusercontent.com/assets/5825883/13593136/c38f853c-e51e-11e5-9500-b25c91040952.png)


i don't see this in my site

**Comment by pigeonflight** [comment]
@drex44 I'm using the latest Plone 5.0.x checkout (buildout.coredev).

**Comment by davidjb** [comment]
Looks like @pigeonflight's screenshot for me in Plone 5.0.3, appears the same in FF 46a2 and Chromium 48. 

The cause is the CSS rule `.icons-on .contenttype-collection { padding-left: 20px!important}` and this is stemming from its inclusion in https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/static/plone-legacy-compiled.css (see the very end of the line).  Removing these style definitions and enabling development mode for the plone-legacy bundle's CSS fixes the issue.

**Comment by davidjb** [comment]
The CSS also affects the Dexterity Content Types page: 
![screenshot from 2016-03-18 13-58-26](https://cloud.githubusercontent.com/assets/1002811/13868150/c65a3ab4-ed11-11e5-92fb-3a2eaab55ccd.png)

(the missing Page/Document icon is another issue which I've already fixed in Dexterity https://github.com/plone/plone.app.dexterity/commit/7586ef1eda2f8e9e8a658880a9de3c2b35e353bd)

What's the process for updating `plone-legacy-compiled.css`?

**Comment by davilima6** [comment]
@davidjb: I think it's this: https://github.com/plone/Products.CMFPlone/blob/master/DEVELOPING_BUNDLES.rst

**Comment by thet** [comment]
Re-stating from https://community.plone.org/t/css-bug-fixing-i-am-about-to-remove-legacy-css-contemplating-if-it-would-be-better-to-override-the-property-in-a-different-file/1809

I had the impression, that ``plone-legacy-compiled.css`` ( https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/static/plone-legacy-compiled.css ) must have been compiled from other source files. But it looks like, it was done manually.
@bloodbare - you added this file - how did you compile it and what are it's sources?

**Comment by bloodbare** [comment]
Hey sorry I missed this thread,

The reason is quite simple:

- there is no map because its not compiled as the compiled resources (requirejs)

- legacy bundles are compiled and stored on the zodb at the first rendering page of a new site in case the "last compilation time" is earlier than the "last time installed a product with legacy bundle". Its designed to compile if you install a new product with jsregistry (that it will be installed in the new registry but we mainteined bbb) then the date of the last installation of legacy bundle will be updated and on the next view css will be "reconcatenated" (remember that legacy it only gets concatenated, not compiled)

- The one that is on Products.CMFPlone/static/plone-legacy-compiled.css it was created just in case there is no product installed on a fresh plone site. That file will be needed to be updated on plone releases. How ? on a really manual way, I just copied the css/js from the resource registry on dev mode and saved on a file a fresh site.

About where it comes from a simple search on plone org on github it looks like it cames from the rtl.css old css that may be outdated and removed since the last time we checkin that file.

So actions to solve that would be to push a new version of it and add it to the release process or just change the site setup to execute a new cookWhenChangingSettings(self.context, legacy) after plone installation to make sure that we have "last fresh version" on the zodb

**Comment by pigeonflight** [comment]
I like the idea of pushing a new version that is added to the release process but I'm scared that this might introduce new regression bugs.

**Comment by bloodbare** [comment]
I puhed [1] a new version of the css and js file based on an actual clean installation.

BTW I remembered that the compilation of the legacy css/js is done when you install a package that has a css/js so on plone installation is done each time you install a plone.

There is still some legacy css (plone.app.event css) that is registered with cssregistry.xml that may be good to migrate to registry.xml.

Regarthing the error I don't see it on coredev 5.1. So I would say we can close this bug

[1] https://github.com/plone/Products.CMFPlone/commit/31e495252ca35b72431f2a255bc2a34b7607455e
